ArcGISRuntimeSDKQt_QMLSamples/Geometry/ClipGeometry/ClipGeometry.qml

```diff
@@ -0,0 +1,198 @@
+// [WriteFile Name=ClipGeometry, Category=Geometry]
+// [Legal]
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 1.4
+import QtQuick.Controls.Styles 1.4
+import Esri.ArcGISRuntime 100.3
+import Esri.ArcGISExtras 1.1
+
+Rectangle {
+  id: rootRectangle
+  clip: true
+  width: 800
+  height: 600
+
+  property real scaleFactor: System.displayScaleFactor
+
+  MapView {
+    id: mapView
+    anchors.fill: parent
+
+    Map {
+      BasemapTopographic {}
+      initialViewpoint: vc
+
+      // Create the envelope for Colorado
+      Envelope {
+        id: colorado
+        xMax: -11362327.128340
+        xMin: -12138232.018408
+        yMin: 4441198.773776
+        yMax: 5012861.290274
+        spatialReference: SpatialReference {
+          wkid: 3857
+        }
+      }
+
+      // Create the dotted outline for the clipping envelopes
+      SimpleLineSymbol {
+        id: redOutline
+        style: Enums.SimpleLineSymbolStyleDot
+        color: "#FFFF0000"
+        width: 3
+      }
```

Comment:
This entire file needs to use the 4-space indent which is standard for Qml files. This is necessary for a consistent look and feel for all samples.

Reply:
What? I thought it was a 2-space indent. Or is that only for C++?

Reply:
I definitely remember changing my editor settings to 2-space because of something Lucas had said.

Reply:
2 spaces for C++, 4 for Qml. You can set language specific editor settings for Qt Creator.